blog/2021-11-30-Building-a-Centralized-Annotated-Database.md

```diff
@@ -0,0 +1,57 @@
+
+
+# FoodDX
+
+FoodDX is a product from Holmusk that is meant to transform the way consumers think and feel about food, it gives real-time feedback on your meals, FoodDX allows users to upload photos of their meal for real-time nutrition scores and food tips. This is enabled via AI food scoring that leverages the companies real world dataset of over 10,000 + images.
+
+ This was made possible by due one of Holmusks previous products Glycoleap which was a product that aimed at helping diabetic patients track their diet and manage their health by letting them get an expert nutritionists advice simply by clicking a photo, and the uploading it to the app, which the nutritionists then graded. The data from this gave us a great boost for FoodDX as it enables us to have access to trusted, high quality data stored in a database. We used this initial data to have in order to train the model, but we also added in completely different data separate from the initial Glycoleap data for the cross verification data, however this was mostly manages via Dropbox where the coaches were given images via Dropbox and similarly shared their scores on the same platform. When FoodDX was up and running, in order to better organize the system, the model rating and all relevant information was stored in a database and similarly, an annotation platform was created for the coaches via retool to accurately to score the image, the food type, and also help, with maintaining the data, as the coach data was saved in a separate datatable.
+
+In the current existing situation, we have a plethora of sources as summarized in the flow diagram below  -
+
+![](/images/blogposts/ML_datapipeline_sources.png)
+
+
+Considering the different sources that we have, we decided to classify the sources into two types -
+
+1. **Static Sources** - These were sources that no longer being continuously updated, since they would no longer be any change to these sources, we decided the data from these sources  needed a one time transfer to the new  database

```

Comment:
```suggestion
1. **Static Sources** - These are sources that are no longer being updated. Since there would no longer be any change to these sources, we decided the data from these sources  needed a one time transfer to the new  database

```

Reply:
Updated, Thanks a lot